lib/vdsm/storage/storageServer.py

```diff
@@ -384,8 +384,7 @@ def _get_backup_servers_option(self):
     def _get_gluster_volinfo(self):
         try:
             superVdsmProxy = supervdsm.getProxy()
-            volinfo = superVdsmProxy.glusterVolumeInfo(self._volname,
-                                                       self._volfileserver)
```

Comment:
I think this was added years ago because not specifying the server could be a problem. Not sure that the gluster cluster can cope with a case when one of the server is down.

Please check git history to understand when and why self._volfileserver was added.

Also how did you test this change? Please describe what was tested - how you simulated the issue and tested that this change solves the issue, and how you tested that this change does not cause regressions in other flows.

Reply:
I found this test:
https://github.com/oVirt/vdsm/blob/master/tests/storage/storageserver_test.py#L268

Using `backup-vol-fileserver`. Maybe the issue you experience can be solved with this configuration option?

Reply:
@mohit84 do you have any suggestions?